src/components/structures/MatrixChat.js

```diff
@@ -181,16 +181,8 @@ export default React.createClass({
             // Parameters used in the registration dance with the IS
```

Comment:
Have you tested the email validation flow with these changes in place? (That's one area we've tended to break...)

Reply:
Yup, several times.

Reply:
just to follow up: I re-tested the two common scenarios just to be absolutely sure it works.

* [x] Creating a new account and verifying that email works
* [ ] Receiving an email invite does not work (see https://github.com/vector-im/riot-web/issues/9816 - this is not an issue unique to this PR)